packages/manager/src/components/DebouncedSearchTextField/DebouncedSearchTextField.tsx

```diff
@@ -77,7 +82,11 @@ const DebouncedSearch = (props: DebouncedSearchProps) => {
             <CircleProgress mini={true} />
           </InputAdornment>
         ) : (
-          <React.Fragment />
+          clearable && (
+            <InputAdornment position="end">
+              <StyledClearIcon onClick={() => setTextFieldValue('')} />
+            </InputAdornment>
```

Comment:
Should this be a button of some kind?

For example:
```suggestion
              <IconButton
                aria-label="Clear"
                onClick={() => setTextFieldValue('')}
                size="small"
                sx={{ padding: 'unset' }}
              >
                <Clear
                  sx={(theme) => ({ color: `${theme.color.grey1} !important` })}
                />
              </IconButton>
```